recipes/xmscore/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "xmscore" %}
+{% set version = "3.0.7" %}
+{% set sha256 = "98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/Aquaveo/{{ name|lower }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  string: py{{ python }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake >=3.0.0
+    - boost =1.67
+    - pybind11 =2.2.4
+    - python {{ python }}
+  host:
+    - python {{ python }}
+    - pip
+  run:
+    - python {{ python }}
+    - numpy
+
+test:
+  source_files:
+    - _package/*.py
+    - _package/**/*.py
+  commands:
+    - python -m unittest discover -v -p *_pyt.py -s ./_package/tests
+  imports:
+    - xms.core
+
+
+about:
+  home: aquaveo.com
+  license: BSD2
+  license_file: LICENSE
+  summary: 'Support library for XMS libraries and products.'
+  doc_url: 'https://aquaveo.github.io/xmscore/'
+  dev_url: 'https://github.com/Aquaveo/xmscore'
+
+extra:
+  recipe-maintainers:
+    - swainn
+    - Tsubashi
+    - gagelarsen
+    - alemon-aquaveo
```

Comment:
We need confirmation that @swainn, @Tsubashi, @gagelarsen, and @alemon-aquaveo agree to be maintainers here. (Just comment below.)

Reply:
I agree

Reply:
I also agree

Reply:
I agree

Reply:
Thanks! We are only missing @alemon-aquaveo.

Reply:
I agree